lib/vrl/tests/src/main.rs

```diff
@@ -324,22 +325,27 @@ fn main() {
     print_result(failed_count)
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
I just don't see the point of creating a struct just for this one function, but will if folks feel strongly enough..

Reply:
Not a fan of that lint either, the difference is pretty inconsequential. I'd even argue to remove it globally.